app/controllers/memberships_controller.rb

```diff
@@ -53,6 +54,7 @@ def membership
     membership_id = params[:id] || params[:membership_id]
     @membership ||= Membership.find(membership_id)
   end
+  helper_method :membership
```

Comment:
Is this necessary here? As I saw it's used in the view, but membership should be available there, using `@membership`.